src/apis/choices.js

```diff
@@ -0,0 +1,5 @@
+import axios from 'axios';
+
+export default axios.create({
+  baseURL: `${process.env.REACT_APP_MERMAID_API_URL}/v1/`
```

Comment:
Does REACT_APP_MERMAID_API_URL and REACT_APP_SUMMARY_API_URL come from the same server?